### PR TITLE
cleans up all the readmes

### PR DESCRIPTION
**Pull request by mchmarny**
Streamlines the verbiage across all the repos, makes the contribution doc go sdk specific, and cleans up examples.

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -213,7 +212,7 @@ secret, err := client.GetSecret(ctx, "store-name", "secret-name", opt)
 
 ## Service (callback)
 
-In addition to this Dapr API client, Dapr go SDK also provides `service` package to bootstrap your Dapr callback services in either gRPC or HTTP. Instructions on how to use it are located [here](./service/Readme.md)
+In addition to this Dapr API client operations, Dapr go SDK also provides `service` package to help you bootstrap Dapr callback services in either gRPC or HTTP. Instructions on how to use it are located [here](./service/Readme.md)
```
Comment (mukundansundar) [inline Readme.md:215]: rephrase `in addition to this Dapr API`

File: service/grpc/Readme.md [hunk 2]
```diff
@@ -6,57 +6,75 @@ Start by importing Dapr go `service/grpc` package:
 daprd "github.com/dapr/go-sdk/service/grpc"
 ```
 
-## Event Handling 
+## Creating and Starting Service 
 
-To handle events from specific topic, first create a Dapr service, add topic event handler, and start the service:
+To create an gRPC Dapr service, first, create a Dapr callback instance with a specific address:
```
Comment (mukundansundar) [inline service/grpc/Readme.md:11]: nitpick: typo `an`

File: service/http/Readme.md [hunk 3]
```diff
@@ -6,13 +6,35 @@ Start by importing Dapr go `service/http` package:
 daprd "github.com/dapr/go-sdk/service/http"
 ```
 
-## Event Handling 
+## Creating and Starting Service 
 
-To handle events from specific topic, first create a Dapr service, add topic event handler, and start the service:
+To create an HTTP Dapr service, first, create a Dapr callback instance with a specific address:
```
Comment (mukundansundar) [inline service/http/Readme.md:11]: nitpick: typo `an`

Comment (mchmarny) [reply service/http/Readme.md:11]: Is that really a type? I'm not an English native speaker but I think the grammar rule says to use `an` before `a` word beginning with a vowel sound, not letter. Or did I get it wrong?

File: service/grpc/Readme.md (hunk 2 above)
Comment (mchmarny) [reply service/grpc/Readme.md]: fixed

File: Readme.md (hunk 1 above)
Comment (mchmarny) [reply Readme.md]: rephrased

File: service/grpc/Readme.md (hunk 2 above)
Comment (mukundansundar) [reply service/grpc/Readme.md]: `To create a gRPC service ... `

File: Readme.md (hunk 1 above)
Comment (mukundansundar) [reply Readme.md]: `In addition to Dapr API client operations,...` remove this

**Review by mukundansundar — APPROVED** [review]

**Pull request merged**